### PR TITLE
Added support for wistar curtain

### PR DESCRIPTION
**Pull request by avp-avp**

### CODE REVIEW

**Comment by mjg59** [comment]
Hi! Unfortunately the library has recently been updated to split it up into more logical components. Would it be possible to rebase this patch on top of the current layout? Sorry it's taken so long to review!

**Comment by felipediel** [comment]
@avp-avp Do you want me to rebase for you so we can have this soon?

File: broadlink/__init__.py [hunk 1]
```diff
@@ -998,6 +999,75 @@ def set_percentage_and_wait(self, new_percentage):
...
+            while current is not None and current > new_percentage:
+                time.sleep(0.2)
+                current = self.get_percentage()
+
+        elif current < new_percentage:
+            self.set_percentage(new_percentage)
+            while current is not None and current < new_percentage:
+                time.sleep(0.2)
+                current = self.get_percentage()
+        self.stop()
```
Comment (felipediel) [inline broadlink/__init__.py:1069]: I don't understand. Do we have to poll the state and stop at the right time?
Can't we just set_percentage() to a position? Why do we have to wait?

**Comment by felipediel** [comment]
Hi @avp-avp. Apologies for the delay. I've rebased this PR to add it to the next release.

Quick question, what is your controller? Maybe [this](https://www.wistarmotor.com/product/control-unit/receiver/wsrc103.html)? Or [this](https://www.wistarmotor.com/product/control-unit/receiver/wsrc201.html)?

 What is the device name on the official Broadlink app?

**Pull request closed without merging**